mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -263,10 +261,10 @@ def list_model_endpoints(
     :param function:  The name of the function to filter by.
     :param labels:    A list of labels to filter by. Label filters work by either filtering a specific value of a label
                       (i.e. list("key==value")) or by looking for the existence of a given key (i.e. "key").
-    :param metrics:   A list of metrics to return for each endpoint. There are pre-defined metrics for model endpoints
-                      such as predictions_per_second and latency_avg_5m but also custom metrics defined by the user.
-                      Please note that these metrics are stored in the time series DB and the results will be appeared
-                      under model_endpoint.spec.metrics of each endpoint.
+    :param metrics:   A list of real-time metrics to return for each endpoint. There are pre-defined real-time metrics
+                      for model endpoints such as predictions_per_second and latency_avg_5m but also custom metrics
+                      defined by the user. Please note that these metrics are stored in the time series DB and t
+                      he results will be appeared under model_endpoint.spec.metrics of each endpoint.
```

Comment:
Right, the metrics are indeed dictionary of dictionaries (as you can see under `ModelEndpointStatus.metrics`). When the user calls `list_model_endpoints` or `get_model_endpoint` API, he can provide a `metrics` list which will filter only the relevant real-time metrics, that mean the metrics under the `ModelEndpointStatus.metrics['real-time']` dictionary. I didn't want to change the `metrics` param name in the API (e.g. to `realtime_metrics`) because I guess that in the future will the user will be able to filter different types of metrics and not just real time (and also because of BC). 
And yes it should work with the new metrics layout as each new metric is a basically a new dictionary under the relevant "parent" dictionary in `ModelEndpointStatus.metrics`.